cypress/integration/finance/ledgers/ledgers.create.spec.js

```diff
@@ -0,0 +1,25 @@
+import Ledgers from '../../../support/fragments/finance/ledgers/Ledgers';
+
+import {
+  Button
+} from '../../../../interactors';
+
+describe('ui-finance: Ledger list search and filters', () => {
+  before(() => {
+    cy.login('diku_admin', 'admin')
+    .then(() => {
+        cy.visit('/finance/ledger');
```

Comment:
put path into TopMenu

Reply:
`cy.visit(TopMenu.ledgerPath)` and update class TopMenu with this